src/util.rs

```diff
@@ -399,7 +399,9 @@ impl U32Ext for u32 {
     }
 }
 
-pub fn build_rs() -> TokenStream {
+pub fn build_rs(settings: &crate::Settings) -> TokenStream {
```

Comment:
maybe it makes sense to pass entire `config` to avoid breaking changes in future?

Reply:
Done!